Resonite Import Helper 0.1.20がコンパイルエラーになってしまう

![Image](https://github.com/user-attachments/assets/4bdbf49c-006b-4789-bfdd-4c419cd59d67)
![Image](https://github.com/user-attachments/assets/2a11198c-6ae3-4803-aa2b-165859bf0e86)

スクリーンショットのように、VCCで liltoon 1.9.0 とResonite Import Helper 0.1.20 を同時にインストールして
Unityプロジェクトの作成をしようとすると、コンパイルエラーが発生することを確認しています。
liltoonのバージョンを1.8.5にすると、このエラーは起こらなくなり、
無事にUnityプロジェクトの作成からResonite用のアバターの作成ができました。

スクリーンショットの環境はWindows11ですが、
macOSでALCOMから同じようにプロジェクトの作成を試みても同じ現象が起こることを確認しています。

お役に立つ情報提供になるなら幸いです。

根本的な原因を特定しました:

```
Packages/io.github.kisaragieffective.resonite-import-helper/Editor/Transform/Environment/AAO/DisableMeshMerge.None.cs(10,44): error CS0246: The type or namespace name 'IPlatformExpander' could not be found (are you missing a using directive or an assembly reference?)
```

根本原因としてはlilToonのバージョンではなく、AAO: Avatar Optimizerのインストール有無に依存します。

--- a/Editor/Transform/Environment/AAO/DisableMeshMerge.None.cs
+++ b/Editor/Transform/Environment/AAO/DisableMeshMerge.None.cs
@@ -7,7 +7,7 @@ namespace KisaragiMarine.ResoniteImportHelper.Transform.Environment.AAO
     /// <summary>
     /// UniGLTF に AAO MergeMesh したアバターを食わせるとメッシュが大幅に壊れる現象を回避するためのパス
     /// </summary>
-    public sealed class DisableMeshMerge : IPlatformExpander
+    public sealed class DisableMeshMerge : IPlatformDependantPreprocessor
     {
         public GameObject PerformEnvironmentDependantShallowCopy(GameObject unmodifiableRoot)
         {

